config/suit/templates/nrf54h20/matter/v1/rad_envelope.yaml.jinja2

```diff
@@ -26,6 +26,8 @@ SUIT_Envelope_Tagged:
         - {{ radio['dt'].chosen_nodes['zephyr,code-partition'].regs[0].size }}
       - - CAND_IMG
         - 0
+      - - CACHE_POOL
```

Comment:
Actually this could be wrapped in 
`{%- if 'CONFIG_SUIT_DFU_CACHE_EXTRACT_IMAGE' not in radio['config'] or radio['config']['CONFIG_SUIT_DFU_CACHE_EXTRACT_IMAGE'] == '' %}`

but this has to be aligned anyway in default SUIT templates, so I will probably do this myself